deployment/services/proxy.ts

```diff
@@ -7,6 +7,25 @@ import { GraphQL } from './graphql';
 import { Observability } from './observability';
 import { Usage } from './usage';
 
+export function deployLabWorker({
+  reverseProxy,
+  app,
+  environment,
+}: {
+  reverseProxy: Proxy;
+  app: App;
+  environment: Environment;
+}) {
+  return reverseProxy.registerInternalProxy({ record: environment.appDns }, [
+    {
+      path: '/worker.js',
```

Comment:
I think you can pass `path` from `index.ts`, because you'll need the `/worker.js` value to build the `PREFLIGHT_SCRIPT_URL` for the app container